gems/sorbet-runtime/test/types/builder_syntax.rb

```diff
@@ -29,6 +29,17 @@ def self.fn(x); x.to_s; end
       assert_equal(true, builder.decl.finalized)
     end
 
+    it 'requires paramms be keyword args' do
+      ex = assert_raises do
+        Class.new do
+          extend T::Sig
+          sig {params(Integer).void}
+          def self.foo; end; foo
+        end
+      end
+      assert_includes(ex.message, "params expects keyword arguments")
+    end
```

Comment:
Could you add a case for `sig {params.void}` as well?

Reply:
https://github.com/sorbet/sorbet/pull/3524/files#diff-3f67f2cd0e5ede62a49719cac03477abf49e2fa975343aa21859a58c8d293dc4R43